frontend/src/employee-frontend/components/unit/TabGroups.tsx

```diff
@@ -57,16 +57,19 @@ export default React.memo(function TabGroups({
             </ContentArea>
           )}
 
-        <ContentArea opaque>
-          <MissingGroupPlacements
-            groups={unitData.groups}
-            missingGroupPlacements={unitData.missingGroupPlacements}
-            backupCares={unitData.backupCares}
-            reloadUnitData={reloadUnitData}
-            permittedPlacementActions={unitData.permittedPlacementActions}
-            permittedBackupCareActions={unitData.permittedBackupCareActions}
-          />
-        </ContentArea>
+        {unitData.missingGroupPlacements &&
+          unitData.missingGroupPlacements.length > 0 && (
```

Comment:
`missingGroupPlacements` is typed as required property so this could be simplified to `unitData.missingGroupPlacements.length > 0 &&`